test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +11,39 @@
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() throws Exception {
+        // empty list
+        assertNotNull();
+
+        // single null element
+        assertHasNull((Object)null);
+
+        // single not null element
+        assertNotNull(1);
+        assertNotNull("a");
+        assertNotNull("");
+
+        // multiple null elements
+        assertHasNull(null, null);
+
+        // multiple not null elements
+        assertNotNull(5, "a");
+        assertNotNull("abc", 3, "");
+
+        // mixed null and not null elements
+        assertHasNull(null, 1);
+        assertHasNull("", 5, null);
+        assertHasNull(3, null, "foo", null);
+    }
+
+    private void assertHasNull(Object... objects) {
```

Comment:
It's good that you followed the style used by the code nearby.
However, in this case, there is no need to abstract the assertTrue(Utils.isAnyNull(...)) as a separate method. Asserting the result of the method itself is fine.
The existing test uses assertAreUnique() in order to prevent Arrays.asList(..) from being used each time.